### PR TITLE
Give Farfetch'd the Stick item in randbats

### PR DESCRIPTION
**Pull request by ascriptmaster**
Just for variety. This might make it worse than whatever items it'd end up with normally, though; it's level scaled high enough as it is.

Also debating whether Slash is actually a good replacement for Return or not. Since it always crits, it has a higher effective Base Power compared to Return (105 BP vs 102! omg!) but Return still has that 50% chance of critting and that factor might make it better overall.

Also note the stealth-edit which only changes some minor comment expressing doubt about a joke long dead

### CODE REVIEW

**Comment by TheImmortal** [comment]
Remove Knock Off because Night Slash is superior on Farfetch'd and it is unnecessary.

File: data/scripts.js [hunk 1]
```diff
@@ -1381,7 +1381,7 @@ exports.BattleScripts = {
 			item = template.requiredItem;
 		} else if (template.species === 'Rotom-Fan') {
 			// this is just to amuse myself
-			// do we really have to keep this
+			// for the sake of the world's balance, this must always stay
```
Comment (Slayer95) [inline data/scripts.js]: The new comment doesn't make sense given the context.
Just remove the <code>// do we really have to keep this</code>, or add the world's balance comment below.

Comment (TheImmortal) [inline data/scripts.js]: Remove the second comment imo.

Comment (ascriptmaster) [inline data/scripts.js]: It was a joke, but fiiiine

File: data/formats-data.js [hunk 2]
```diff
@@ -713,7 +713,7 @@ exports.BattleFormatsData = {
 		tier: "OU"
 	},
 	farfetchd: {
-		randomBattleMoves: ["bravebird","swordsdance","return","leafblade","roost","knockoff"],
+		randomBattleMoves: ["bravebird","swordsdance","retrun","leafblade","roost","nightslash"],
```
Comment (TheImmortal) [inline data/formats-data.js]: nice typo

**Comment by ascriptmaster** [comment]
Is the syntax I used for checking for Farfetch'd correct? I can't really test it right now

**Comment by TheImmortal** [comment]
Yes, I tested it.

File: data/formats-data.js (hunk 2 above)
Comment (ascriptmaster) [inline data/formats-data.js]: what is spilling, and how do i do it good

**Comment by ascriptmaster** [comment]
I'll be throwing in another pullreq updating Hoopa-Unbound later to allow mixed sets, since Hyperspace Fury is a Physical move. Bulbapedia says that ORAS move tutors give it the Elemental Punches and Drain Punch, and it has Zen Headbutt for STAB.

**Comment by Zarel** [comment]
I'd say Slash is better. Autocrit is nice against Intimidate or against Iron Defense et al.

**Comment by Zarel** [comment]
And randbat is all about seeing things you don't normally see. I'd rather lean towards "gimmicky" than "safe and boring".

**Comment by TheImmortal** [comment]
There's only a handful of def boosters in Random Battle (Cotton Guard). Also, Leaf Blade and Night Slash autocrit for those cases.

**Comment by TheImmortal** [comment]
Isn't that Challenge Cup.

**Comment by Zarel** [comment]
Oh, Return has a 50% crit chance? Definitely go with Return.

**Comment by ascriptmaster** [comment]
Yeah, Stick gives +2 critical, which in Gen 6 means all moves have 50% chance of crit.

High-crit moves like Leaf Blade/Night Slash give +1 critical, and at +3 critical you have autocrit.

At any rate, a base 102 attack that has a 50% chance of critting is better than an attack that always crits, but after the crit has only 107 base power.

**Pull request merged**